proposal.tex

```diff
@@ -37,6 +48,12 @@ \subsection{BPS Batch }\label{sec:bpsbatch}
 This would also cover DMS-REQ-0119, DMS-REQ-0125, DMS-REQ-0128, DMS-REQ-0123, DMS-REQ-0127.
 
 
+\subsubsection{Non Rubin code} \label{sec:othercode}
+Users may wish to run arbitrary code on Rubin images.
+It should be trivial to provide a butler recipe to get an image or set of images to the users space and allow them to run any code which takes FITS images. Done in a container this could be done for a large volume of images - it would have to get past  the resource allocation committee of course.
```

Comment:
"trivial" is going a bit far. We'd have to provide some infrastructure to allow them to define the connections via config, for example, along with the the quantum DataId. Doing a one-off PipelineTask for a specific set of connections that calls arbitrary binaries is not hard, making it a general facility for all possible connections (inputs and outputs) needs thought.

Reply:
But we do not need a general facility - we discourage this not make it easy .. but i put possible instead of trivial and noted manual work needed not a general facility.